nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/main/java/org/apache/nifi/provenance/lucene/DocsReader.java

```diff
@@ -47,9 +46,6 @@
 public class DocsReader {
     private final Logger logger = LoggerFactory.getLogger(DocsReader.class);
 
-    public DocsReader(final List<File> storageDirectories) {
-    }
-
```

Comment:
@apiri The versioning scheme is defined at https://cwiki.apache.org/confluence/display/NIFI/Version+Scheme - it essentially says that all extension points that adhere to API version X.Y.* must also work in version X.(Y+1).*. I.e., minor changes will not break backward compatibility of extension point API or those objects that can be reached by extension points via the API.